docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -54,194 +24,171 @@ export COBALT_PIPELINE_NAME="Cobalt CICD Pipeline"
     This step helps setup your Azure Devops repo with Cobalt Advocated Pattern Templates that matter to you. These are common instructions that are needed for any audience interested in using Cobalt for infrastructure automation.
 
     * Create a new project
-        * Sign-in to Azure DevOps (https://azure.microsoft.com/en-us/services/devops/)
-        * Select New project and create a name. (ex. Cobalt-Contoso)
-
-            ![New Project](https://user-images.githubusercontent.com/10041279/63442791-4f868600-c3f9-11e9-91f3-c959654f5a1c.png)
+        1. Sign-in to Azure DevOps (https://azure.microsoft.com/en-us/services/devops/)
+        2. Create new project under Organization. First create new Organization if it don't exists already. (ex. Cobalt-Contoso)
 
-        * Select Create
+            ![alt text](./images/Org.png "Create Organization")
+            ![alt text](./images/project.png "Create Project")
 
-> The following CLI command(s) can be run as an alternative to using the portal-based instructions:
+    * Create new repository by fetching source code from the master branch of Cobalt's open-source github project
```

Comment:
It would be helpful context for the reader to explain why the whole cobalt repo needs to be imported.

Reply:
Added the following line. "This will allow you to host and grow your own repo for Cobalt Infrastructure Templates."